back/app/services/community_monitor_service.rb

```diff
@@ -11,6 +11,12 @@ def project_id
     settings.dig('community_monitor', 'project_id')
   end
 
+  def project
+    return nil unless enabled?
+
+    Project.find(project_id)
+  end
+
```

Comment:
I'm not quite sure what this does? :)

Reply:
This finds the Community monitor project using the project id stored in settings